components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/configuration/models/APIMConfigurations.java

```diff
@@ -91,6 +91,17 @@
     @Element(description = "SDK Generation Language Configurations")
     private SdkLanguageConfigurations sdkLanguageConfigurations = new SdkLanguageConfigurations();
 
+    @Element(description = "SDK Generation Language Configurations")
```

Comment:
The description here is a mistake. Shall we change it?

Reply:
Thanks. Completely missed it. I'll correct.